spec/javascripts/unit/AnalyticsSpec.js

```diff
@@ -187,32 +187,74 @@ describe("GOVUK.Analytics", function () {
+      window.GOVUK.GDM.analytics.virtualPageViews.init();
       expect(window.ga.calls.first().args).toEqual([ 'send', 'pageview', { page: 'http://example.com/vpv' } ]);
       expect(window.ga.calls.count()).toEqual(1);
     });
 
 
-      it("Should add '/vpv/' to url before question mark", function () {
-        $analyticsString = $('<div data-analytics="trackPageView" data-url="http:/testing.co.uk/testrubbs?sweet"/>');
-        $(document.body).append($analyticsString);
-        window.GOVUK.GDM.analytics.virtualPageViews();
-        expect(window.ga.calls.first().args[2]).toEqual({page: "http:/testing.co.uk/testrubbs/vpv?sweet"});
-      });
+    it("Should add '/vpv/' to url before question mark", function () {
+      $analyticsString = $('<div data-analytics="trackPageView" data-url="http:/testing.co.uk/testrubbs?sweet"/>');
+      $(document.body).append($analyticsString);
+      window.GOVUK.GDM.analytics.virtualPageViews.init();
+      expect(window.ga.calls.first().args[2]).toEqual({page: "http:/testing.co.uk/testrubbs/vpv?sweet"});
+    });
+
+    it("Should add '/vpv/' to url at the end if no question mark", function () {
+      $analyticsString = $("<div data-analytics='trackPageView' data-url='http://example.com'/>");
+      $(document.body).append($analyticsString);
+      window.GOVUK.GDM.analytics.virtualPageViews.init();
+      expect(window.ga.calls.first().args[2]).toEqual({page: "http://example.com/vpv"});
+    });
 
-      it("Should add '/vpv/' to url at the end if no question mark", function () {
-        $analyticsString = $("<div data-analytics='trackPageView' data-url='http://example.com'/>");
-        $(document.body).append($analyticsString);
-        window.GOVUK.GDM.analytics.virtualPageViews();
-        expect(window.ga.calls.first().args[2]).toEqual({page: "http://example.com/vpv"});
-      });
+    it("Should trigger virtual page view on filter selection", function () {
+      $analyticsString = $(
+        '<form id="js-dm-live-search-form">' + 
+            '<div class="dm-filters">' +
+              '<div class="options-container" id="example-filters-group">' +
+                '<div class="js-auto-height-inner">' +
+                  '<label for="filter-option-1">' +
+                    '<input name="filter-option-1" value="filter-option-1-value" id="filter-option-1" type="checkbox" aria-controls="">' +
+                    'filter-option-1-label' +
+                  '</label>' +
+                '</div>' +
+              '</div>' +
+            '</div>' +
+          '</form>'
+      );
+      $(document.body).append($analyticsString);
+      window.GOVUK.GDM.analytics.virtualPageViews.init();
+      $('#filter-option-1').click();
+      expect(window.ga.calls.first().args[2]).toEqual({page: "/g-cloud/filters/example-filters-group/filter-option-1/filter-option-1-value/vpv"});
+    });
+
+    it("Should not trigger virtual page view on filter selection", function () {
```

Comment:
I understood the first test, but this one seems identical (except that we don't expect it to trigger vpv, and it's not immediately obvious why?)